src/components/ProjectDashboard/components/CyclesPayoutsPanel/components/CyclesPanelTooltips.tsx

```diff
@@ -0,0 +1,30 @@
+import { Trans } from '@lingui/macro'
+
+export const cycleStatusTooltip = (
+  <Trans>Locked cycles means... //waiting on Filip</Trans>
```

Comment:
Lets remove the fillip comment/s so its still prod ready. zfilip can change later